### PR TITLE
Documentation for the 'priority_studies' feature in portal.properties

### PR DESCRIPTION
**Pull request by sowmiyaa-kumar**
See: https://github.com/cBioPortal/cbioportal/pull/10441/files?short_path=1b90465#diff-1b90465f4a183b71cc5bccd0c0ec2af01877c09e97a12dc6d42a1c6549df2219

### CODE REVIEW

File: docs/deployment/customization/portal.properties-Reference.md [hunk 1]
```diff
@@ -714,3 +714,10 @@ The studies to be downloaded need to be compressed with the extension ``tar.gz``
 * ``study_list.json`` resides ``https://cbioportal-datahub.s3.amazonaws.com/study_list.json``
   * ``[ "acbc_mskcc_2015", "acc_2019"]`` Example of contents
 * ``acbc_mskcc_2015.tar.gz`` resides `https://cbioportal-datahub.s3.amazonaws.com/acbc_mskcc_2015.tar.gz`
+
+# Prioritized studies on study selector view 
+By default, the studies loaded onto a local cBioPortal instance are organized alphabetically on the basis of their category (i.e. Breast >> Other).
```
Comment (MatthijsPon) [inline docs/deployment/customization/portal.properties-Reference.md:719]:
```suggestion
By default, the studies loaded into a local cBioPortal instance are organized based on their cancer type (i.e. Breast >> Other).
```

File: docs/deployment/customization/portal.properties-Reference.md [hunk 2]
```diff
@@ -714,3 +714,10 @@ The studies to be downloaded need to be compressed with the extension ``tar.gz``
...
+```
+priority_studies=
+```
+The value to this variable will force study ids and categories to the top of the study selector view. The format for the string should be category1#study1a,study1b,study1c;category2#study2 (for e.g. Other#msk_impact_2017), where the ``category`` can be any custom category and the ``study`` should be the study ID of the required uploaded study.
```
Comment (MatthijsPon) [inline docs/deployment/customization/portal.properties-Reference.md:723]:
```suggestion
The value of this variable create a custom category with studies on the top of the study selector view. The format for the string should be category1#study1a,study1b,study1c;category2#study2 (e.g., PanCancer Studies#msk_impact_2017), where the ``category`` can be any string and the ``study`` should be the study ID of the required uploaded study. 
```
Using the public portals prio will improve clarity, I think :)

**Review by MatthijsPon — CHANGES_REQUESTED** [review]
Thanks for looking into this @sowmiyaa-kumar! Just some small stuff to improve clarity

**Review by inodb — APPROVED** [review]
LGTM!

**Pull request merged**